lib/reek/cli/utilities.rb

```diff
@@ -0,0 +1,12 @@
+module Reek
+  module CLI
+    # CLI utilities
+    module Utilities
+      def self.require_silently
+        old_verbose, $VERBOSE = $VERBOSE, nil
+        yield if block_given?
```

Comment:
I agree, but AFAIS using block_given? even in such cases is pretty idiomatic.
I can remove it if you insist but from personal point of view I'd leave it in...